swap_meet/vendor.py

```diff
@@ -1,2 +1,110 @@
 class Vendor:
-    pass
\ No newline at end of file
+    '''Creating Vendor class that has an inventory attribute which is an optional argument. Will be able to add, remove, look at
+    item categories and item ages and be able to swap items with friends.'''
+    def __init__(self, inventory=None): # O(1)
+        if inventory is None: # O(1)
+            inventory = [] 
+        self.inventory = inventory 
+
+
+    def add(self, item:str): # O(1)
+        '''
+        Returns item that has been appended to inventory list.
+        Parameters: item '''
+        self.inventory.append(item) # O(1)
+        return item 
+    
+
+    def remove(self, item:str): # O(1)
+        '''
+        Returns item that has been removed from inventory list.
+        Parameters: item''' 
+        if item in self.inventory: # O(1)
+            self.inventory.remove(item) # O(1) 
```

Comment:
List removal is O(n) with n being the list of the inventory here; this is because to remove an item, Python would need to do a search over the list, find it, then remove it.